examples/perfect_catke_calibration.jl

```diff
@@ -143,7 +143,7 @@ eki = EnsembleKalmanInversion(calibration;
 
 # and perform few iterations to see if we can converge to the true parameter values.
 
-iterate!(eki; iterations = 20)
+iterate!(eki; iterations = 30)
```

Comment:
nooooo

Reply:
can we reduce the noise rather than iterating more?

Reply:
That sounds like a good idea!

Reply:
We can actually go back to 20. I was playing around and forgot to change it back. See the final figure in the docs. https://clima.github.io/OceanTurbulenceParameterEstimation.jl/dev/literated/perfect_catke_calibration/